### ISSUE TITLE
Bug regarding Smart Response option

### ISSUE THREAD

**Issue by Aarti506** [dannagle/PacketSender#216]
#### What OS?

- [ ] Linux (18.04)

#### Description of issue
Smart response is responding only once , even though i am sending a same data packet to the packet sender.
I am receiving the response only once.Later it is stopped responding.

**Comment by febalci** [comment]
OSX 11 Big Sur and OSX Catalina; same problem.

**Comment by febalci** [comment]
Duplicate of #108

**Comment by Aarti506** [comment]
This issue is not found while working with UDP

**Comment by dannagle** [comment]
Fixed in development branch.
